Program.cs

```diff
@@ -58,11 +60,58 @@ private static void ExtractLicense(Product product)
                     if (match.Success)
                     {
                         Console.WriteLine($"Found key for {product.Name}: {match.Captures[0]}");
+                        Console.WriteLine("\nDo you want to save it? (Yes/No)");
+                        string chooseIfSave = "";
+
+                        chooseIfSave = Console.ReadLine();
+
+                        if (chooseIfSave == "Y" || chooseIfSave == "yes" || chooseIfSave == "Yes")
+                        {
+                            TxtFile("Found key for", product.Name, match.Captures[0].ToString());
+                        }
+                        if (chooseIfSave == "N" || chooseIfSave == "no" || chooseIfSave == "No")
+                        {
+                            Console.Clear();
+                        }
                     }
                 }
             }
             catch (Exception) { }
         }
-    }
-}
 
+        private static void ChoosePath(out string path)
```

Comment:
This would ideally come from the -o command line argument, and you wouldn't need to have the user specify it interactively